libraries/chef_f5.rb

```diff
@@ -13,6 +23,16 @@ def node_is_missing?(name)
       Array(response[:item]).grep(/#{with_partition name}/).empty?
     end
 
+    def node_is_enabled?(name)
+      response = api.LocalLB.NodeAddressV2.get_object_status(name)
+
+      return response[:enabled_status][0] == EnabledStatus::ENABLED_STATUS_ENABLED
```

Comment:
A small style note -- the explicit `return` is unnecessary.

Reply:
Good catch!